plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -198,6 +198,26 @@ export function AboutContent(props: AboutContentProps) {
           <Chip key={t} size="small" label={t} />
         ))}
       </AboutField>
+      <AboutField
+        label="Labels"
+        value="No Labels"
```

Comment:
Was thinking that perhaps it makes sense to hide this completely if there are no labels? Afaik it's fairly common to run a catalog without any labels.

Reply:
Hi,
Thanks @Rugvip for the suggestion. I have made the necessary changes in the code and updated it.